DeploymentCloud/Deployment.DataX/common.parameters.txt

```diff
@@ -32,6 +32,21 @@ databricksClusterNodeType=Standard_DS3_v2
 #Databricks SKU
 databricksSku=premium
 
+# HDInsight cluster version.
+HDInsightVersion=3.6
+
+# The spark version of the HDInsight. For HDInsight4.0, it can be 2.4 or 2.3. For HDInsight3.6, it should be 2.3
```

Comment:
Have we tested 2.3 on HDinsight 4.0 ?

Reply:
yes, i have tested this scenario as well. This is the resource group https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/a6c045ab-34ea-44a7-a4ff-a3926be9d2f4/resourceGroups/HDI4Spark23AutoScale1/overview